src/redux/transformers/index.ts

```diff
@@ -38,53 +38,53 @@ const getRolesForUserAndParentDomains = (
   return Array.from(roleSet);
 };
 
-const getCombinedRolesForDomains = (
-  domainIds: number[],
-  domainRoles: DomainRoles[],
-) => {
-  return Array.from(
-    domainRoles
-      .filter(({ domainId }) => domainIds.includes(domainId))
-      .reduce((roleSet, domainRole) => {
-        domainRole.roles.forEach((role) => roleSet.add(role));
-        return roleSet;
-      }, new Set<ColonyRole>()),
-  );
-};
+// const getCombinedRolesForDomains = (
```

Comment:
Why is this commented out?

Reply:
The way roles work is supposed to change so commenting it in preparation for that work and as a reminder to not use this for now.